### PR TITLE
chore(docs): assorted improvements to the readme

### PR DESCRIPTION
**Pull request by arjunattam**
Restructures the readme to lead with value prop, adds a new example to show more capabilities, and minor word smithing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,32 +3,47 @@
...
-|          | ver | Linux | macOS | Win |
+|          | Version | Linux | macOS | Windows |
 |   ---:   | :---: | :---: | :---:  | :---: |
 | Chromium| 81.0.4044 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
 | WebKit | 13.0.4 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| Firefox |73.0b3 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-- Headless is supported for all the browsers on all platforms.
+| Firefox | 73.0b3 | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+
+Headless is supported for all the browsers on all platforms.
```
Comment (aesyondu) [inline README.md]: What about Internet Explorer?

File: README.md [hunk 2]
```diff
@@ -110,65 +104,100 @@ const { firefox } = require('playwright');
...
   })
   console.log(dimensions);
-
   await browser.close();
 })();
 ```
 
-## Contributing
+#### Intercept requests and wait for navigation
+Playwright can intercept network requests from a page to [modify or stub](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-request) them. Playwright scripts can also wait for [page navigations](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitfornavigationoptions), [selectors](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options), [network responses](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforresponseurlorpredicate-options), and [arbitrary predicates](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforfunctionpagefunction-options-args). This snippet intercepts network requests from a Firefox page, interacts with the page and waits for a navigation.
```
Comment (pavelfeldman) [inline README.md]: nit: This should link to the `page.route` once we push 0.11

File: README.md [hunk 3]
```diff
@@ -110,65 +104,100 @@ const { firefox } = require('playwright');
...
+```js
+const { firefox } = require('playwright');
+
+(async () => {
+  const browser = await firefox.launch({ headless: false });
+  const context = await browser.newContext();
+  const page = await context.newPage('http://todomvc.com');
+
+  // Intercept all network requests and log their URL
+  page.route('**', request => {
```
Comment (pavelfeldman) [inline README.md]: Users on 0.10 will paste it and it won't work. Let's publish this with 0.11

File: README.md [hunk 4]
```diff
@@ -110,65 +104,100 @@ const { firefox } = require('playwright');
...
+Playwright can intercept network requests from a page to [modify or stub](https://github.com/microsoft/playwright/blob/master/docs/api.md#class-request) them. Playwright scripts can also wait for [page navigations](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitfornavigationoptions), [selectors](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforselectorselector-options), [network responses](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforresponseurlorpredicate-options), and [arbitrary predicates](https://github.com/microsoft/playwright/blob/master/docs/api.md#pagewaitforfunctionpagefunction-options-args). This snippet intercepts network requests from a Firefox page, interacts with the page and waits for a navigation.
 
-Check out our [contributing guide](https://github.com/microsoft/playwright/blob/master/CONTRIBUTING.md).
 
-## FAQ
+```js
+const { firefox } = require('playwright');
+
+(async () => {
+  const browser = await firefox.launch({ headless: false });
```
Comment (pavelfeldman) [inline README.md]: why headful?

File: README.md [hunk 5]
```diff
@@ -110,65 +104,100 @@ const { firefox } = require('playwright');
...
 
-- [*ergonomics*] We continue the trend set with Puppeteer and provide ergonomically-sound APIs for frames, workers, handles, etc.
+We recognize WebDriver as a universal standard for the web automation and testing. At the same time we were excited to see Puppeteer affect the WebDriver agenda and steer it towards a bi-directional channel. We hope that Playwright can take it further and pioneer support for numerous PWA features across the browsers as they emerge.
 
-- [*reliability*] With Playwright, we encourage `setTimeout`-free automation. The notion of the wall time is incompatible with the operation in the cloud / CI. It is a major source of flakiness and pain and we would like to provide an alternative. With that, Playwright aims at providing sufficient amount of events based on the browser instrumentation to make it possible.
+- **Reliability**: With Playwright, we encourage `setTimeout`-free automation. The notion of the wall time is incompatible with cloud operation and is a major source of flakiness. Playwright provides sufficient amount of events based on the browser instrumentation to eliminate flakiness.
```
Comment (pavelfeldman) [inline README.md]: Again, re-iterate **Capable** and **Reliable** ? Also, with the top-level items I feel like we repeat each other here.

File: README.md (hunk 1 above)
Comment (arjun27) [reply README.md]: at this point, Playwright does not support any other browsers. we can always update the readme if/when that changes

File: README.md (hunk 5 above)
Comment (arjun27) [reply README.md]: thanks for the review! updating the pitch to be more clear and removing redundancies with the FAQs

File: README.md (hunk 3 above)
Comment (arjun27) [reply README.md]: updated snippet to work with 0.10 as we are doing for the other snippets

File: README.md (hunk 2 above)
Comment (arjun27) [reply README.md]: opting to keep the readme relevant for 0.10 for now, and bring 0.11 changes together when we release

**Comment by arjun27** [comment]
ready for review, ptal

**Comment by pavelfeldman** [comment]
Ouch, I did not review it and now it won't apply. Could you rebaseline? (sorry)

**Pull request closed without merging**